test/test_jit_fuser_te.py

```diff
@@ -1152,20 +1152,48 @@ def rand(dtype, device="cuda"):
             torch.int16,
             torch.int32,
             torch.int64,
-            torch.float16,
-            torch.bfloat16,
+            # torch.float16,
```

Comment:
Nit: maybe add a comment whether we want these to be supported at all?

Reply:
Oh, yeah, float16 should be coming in soon-ish.  The commented lines are a transient state for this stack :-)